### PR TITLE
full functionality by task + bootstrap for main pages

### PR DESCRIPTION
**Pull request by alextimonov**
(cherry picked from commit 3462248)

### CODE REVIEW

File: src/main/java/ua/goit/timonov/enterprise/module_6_2/model/Order.java [hunk 1]
```diff
@@ -51,6 +51,8 @@
     @JsonIgnore
     private List<Dish> dishes;
 
+//    private int nDishes;
```
Comment (schaffe) [inline src/main/java/ua/goit/timonov/enterprise/module_6_2/model/Order.java]: do you need this comment?

File: src/main/java/ua/goit/timonov/enterprise/module_6_2/dao/jdbc/JdbcOrderDAO.java [hunk 2]
```diff
@@ -85,7 +82,7 @@ private boolean orderIsOpen(int orderId) {
     }
 
     // deletes order from auxiliary table
-    @Transactional(propagation = Propagation.MANDATORY)
```
Comment (schaffe) [inline src/main/java/ua/goit/timonov/enterprise/module_6_2/dao/jdbc/JdbcOrderDAO.java:88]: Haven`t you committed it before?

File: src/main/java/ua/goit/timonov/enterprise/module_6_2/model/Order.java (hunk 1 above)
Comment (alextimonov) [reply src/main/java/ua/goit/timonov/enterprise/module_6_2/model/Order.java]: fixed

File: src/main/java/ua/goit/timonov/enterprise/module_6_2/dao/jdbc/JdbcOrderDAO.java (hunk 2 above)
Comment (alextimonov) [reply src/main/java/ua/goit/timonov/enterprise/module_6_2/dao/jdbc/JdbcOrderDAO.java:88]: Yes, I have. Maybe some mistakes during cherry-pick merging

**Pull request merged**